index.js

```diff
@@ -295,8 +334,13 @@ function parse (args, opts) {
           } else {
             setArg(key, defaultValue(key))
           }
+        } else {
+          unmatched.push(key)
         }
       }
+      if (unmatched.length > 0) {
+        argv._.push(maybeCoerceNumber('_', ['-', ...unmatched].join('')))
```

Comment:
the values are already processed, right?

Reply:
Fair point.  I'll remove the number parsing here too.